CONTRIBUTING.md

```diff
@@ -101,12 +102,14 @@ the following tasks:
 4. Open a pull request to
    [`fastify/website:HEAD`](https://github.com/fastify/website/pulls) adding
    yourself to the
-   [team.yml](https://github.com/fastify/website/blob/HEAD/src/website/data/team.yml)
+   [team.yml](https://github.com/fastify/website/blob/HEAD/static/data/team.yml)
    file. This list is also sorted alphabetically so make sure to add your name
    in the proper order. Use your GitHub profile icon for the `picture:` field.
 5. The person that does the onboarding must add you to the [npm
    org](https://www.npmjs.com/org/fastify), so that you can help maintaining the
    official plugins.
+6. Optionally, the person can ask to the Open Collective admins (lead team) to
+   figure out as a collaborator in the Fastify's Open Collective page.
```

Comment:
Bit lost as to what this is trying to convey?

Reply:
Here: https://opencollective.com/fastify at the bottom there is the `Out team` section and you need to be added to be there